### PR TITLE
Fixing issue #170

### PR DESCRIPTION
**Pull request by bmenne-dspace**

### CODE REVIEW

**Review by msuevern — APPROVED** [review]

**Comment by chrbertsch** [comment]
can we merge this before tagging beta.1?

**Comment by bmenne-dspace** [comment]
> can we merge this before tagging beta.1?

@chrbertsch yes dSPACE will take care of integrating https://github.com/modelica/fmi-ls-bus/issues/170 and https://github.com/modelica/fmi-ls-bus/issues/124 before creating the beta release. As discussed in the last FMI Design Meeting I will bring some slides for the next FMI Design Meeting to get a decision for https://github.com/modelica/fmi-ls-bus/issues/170.  

After this we can build the beta release and also fixed all issues concerning this milestone. 

@msuevern fyi

**Review by chrbertsch — CHANGES_REQUESTED** [review]
As the alpha.1 has already been tagged before with a different versioning mechanis, should we not go for version="1.0.0-beta.1"?

**Comment by bmenne-dspace** [comment]
@chrbertsch We can do it that way. I thought about it too and wasn't sure whether I should set it to alpha first and then set it to beta via the other issue (creating beta.1) or not. But we can also set it to beta.1 directly. I don't think there's really a right solution here.

**Review by chrbertsch — APPROVED** [review]

File: docs/4____network_abstraction.adoc [hunk 1]
```diff
@@ -281,27 +281,23 @@ The following table lists the names of MIME types used for the Tx/Rx data variab
 |MIME type
 |Description
 
-|org.fmi-standard.fmi-ls-bus.v1.can
-|Binary variables simulating the automotive CAN network including CAN, CAN FD and CAN XL
+|org.fmi-standard.fmi-ls-bus.can; version="1.0.0-beta.1"
```
Comment (PTaeuberDS) [inline docs/4____network_abstraction.adoc:284]: We should add the media type here, e.g., "application":
application/org.fmi-standard.fmi-ls-bus.can; version="1.0.0-beta.1"

Comment (bmenne-dspace) [reply docs/4____network_abstraction.adoc:284]: > We should add the media type here, e.g., "application": application/org.fmi-standard.fmi-ls-bus.can; version="1.0.0-beta.1"

Yes we can do it if everybody is fine with it. Maybe we can have a short discussion this afternoon, then I can fix it and merge that PR?

**Review by PTaeuberDS — APPROVED** [review]

**Comment by chrbertsch** [comment]
FMI Design Webmeeting
Bendikt: if we exchange FMUs win alpha or beta stage, we have to know ...
Klaus: you can read from manifest file, if "alpha" or "beta"
Pierre: I am against multiple versioning semantics ... I would spell "v1.0.0-alpha.1" out 
Benedikt: we will not change the version without a change in the operations.
Pierre: we should split the standard document (perhaps later)... Then we have the version number of the sub-documents 
Klaus: Can we go through the different scenarios?
Pierre: we will release a v1.0.0 for the whole document ... Later we could split in different sub-standards 

Merge this PR!

**Pull request merged**